### ISSUE TITLE
Quoted aliases (with spaces)

### ISSUE THREAD

**Issue by tbrielle** [JSQLParser/JSqlParser#168]
Tested with JSQLParser 0.9.3 (and JSQLParser 0.9.4-SNAPSHOT at date of writing)

The following queries are valid in MSSQL :

```sql
    select top 10 mycolumn as [xxx yyy] from mytable;
    select top 10 mycolumn as 'xxx yyy' from mytable;
```
However the parsing of the last one fails with the exception:

    net.sf.jsqlparser.parser.ParseException: Encountered " <S_CHAR_LITERAL> "\'xxx yyy\' "" at line 1, column 27.

**Comment by wumpz** [comment]
That is right. Your SQL is not valid. Databaseobject names are not quoted using single quotation marks. Or is there a database that supports this? JSqlParser supports [colname], "colname", and `colname` (apostroph used by MSAccess).

**Comment by tbrielle** [comment]
It seems to be supported in TSQL

**Comment by wumpz** [comment]
Agreed. Never saw that before, but it is very non standard und a deprecated feature:

https://msdn.microsoft.com/en-us/library/bb510662%28v=sql.110%29.aspx
"String literals as column aliases"

Additional you could create so much confusion. 

select 'test' from mytable

Is test a column or a string literal.

JSqlParser will not support this.

Tobias

**Comment by tbrielle** [comment]
OK. Thank you.

**Comment by wumpz** [comment]
solved #583
